frontends/web/package.json

```diff
@@ -6,43 +6,39 @@
     "gen-ts-css": "tcm src --camelCase dashes",
     "build": "yarn run gen-ts-css && preact build --no-sw --no-esm --no-inline-css --no-prerender --verbose",
     "dev": "yarn run gen-ts-css && preact watch",
-    "lint": "yarn run gen-ts-css && eslint --max-warnings 0 --cache --ignore-path ../../.gitignore '**/*.{js,jsx}'",
-    "postlint": "yarn run gen-ts-css && tslint -e 'src/**/*.css.d.ts' 'src/**/*.ts?(x)'",
+    "prelint": "yarn run gen-ts-css",
+    "lint": "eslint --max-warnings 0 --cache --ignore-path ../../.gitignore 'src/**/*.{js,jsx,ts,tsx}'",
```

Comment:
Would you mind adding `--format unix`? This is the format in which every editor can automatically jump to the source code at the error, as it is always `filename:linenumber <error message>`.